README.md

```diff
@@ -170,6 +170,20 @@ pgdb:UID=mauro;PWD=xxx;DSN=pmf
 myver:UID=mauro;PWD=xxx;DSN=vmf
 mysql:USER=mauro;PASSWORD=xxx;DSN=mmf
 ```
+### Alternative connection definition methods
+As we have seen in the previous section we can define multiple "Connection Identifiers" in the "Connection Identifier Database" file which is located by default in ``/usr/local/etc/dblinks.cids`` and then use the ``cid`` parameter to pick our connection:
+
+```sql
+SELECT DBLINK(USING PARAMETERS cid='myconnecction', query=...) ...
+```
+
+There are other methods you can use to define the connection parameters. The **first alternative** is to define all parameters using ``connect_secret``:
+
+```sql
+SELECT DBLINK(USING PARAMETERS connect_secret='UID=mauro;PWD=secret;...', query=...) ...
+```
+
+This way you don't have to create a the dblink.cids database however defining the connection parameters in the command line is not safe before Vertica 12.0.4.  All queries are recorded under ``v_monitor.query_requests`` and in the log file, and that can expose passwords in the ``connect_secret`` parameter.
```

Comment:
Is it really as simple as this? Do we not need to create session parameters?

Reply:
Not sure to understand why we want change the **UDx parameter** "connect" with another **UDx parameter** named "connect_secret".

Reply:
Yes, it is as simple as this in 12.0.4.
I added back the session parameter, so that is an option as well, but it's only there for convenience now.